src/vivarium_public_health/risks/base_risk.py

```diff
@@ -270,10 +268,11 @@ def get_exposure_pipeline(self, builder: Builder) -> Pipeline:
         return builder.value.register_value_producer(
             self.exposure_pipeline_name,
             source=self.get_current_exposure,
-            requires_columns=required_columns,
-            requires_values=[
-                self.propensity_pipeline_name,
-                self.exposure_distribution.parameters_pipeline_name,
+            component=self,
+            required_resources=required_columns
+            + [
+                self.propensity,
+                f"{self.risk}.exposure_parameters",
```

Comment:
This should be the actual pipeline not the name of it `f"{self.risk}.exposure_parameters"`